wire-ios/WireUITests/Pages/PageModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+
+class PageModel {
```

Comment:
question: are you following Page Model Object like here: https://swiftwithmajid.com/2021/03/24/ui-testing-using-page-object-pattern-in-swift/ ?

Reply:
Yes although not fully and I haven't made all the pages yet (some are still squished together when they normally would be separate pages). For example one of the the things I'm trying to figure out locally is how to move the expectation into the page objects themselves in a way which doesn't break the paradigm